resources/helm/overlays/istio-telemetry/prometheus/templates/configmap.yaml

```diff
@@ -34,11 +34,15 @@ data:
       - source_labels: [__meta_kubernetes_pod_container_port_name]
         action: keep
         regex: '.*-envoy-prom'
-      - source_labels: [__address__, __meta_kubernetes_pod_annotation_prometheus_io_port]
+      - source_labels: [__meta_kubernetes_pod_ip]
```

Comment:
How come you change this?

Reply:
It's much easier to write a regex for a pod IP than for an address, because the address contain a port. In this case we don't need `__meta_kubernetes_pod_annotation_prometheus_io_port`, because 15090 is used when `enablePrometheusMerge` is disabled.